2020/hamming-codes/hungarian/sentence_translations.json

```diff
@@ -1414,7 +1414,7 @@
  },
  {
   "input": "To see what I mean, come join me in part two.",
-  "translatedText": "Hogy lásd, mire gondolok, gyere velem a második részbe.",
+  "translatedText": "Hogy lásd, mire gondolok, tarts velem a második részbe.",
```

Comment:
```suggestion
  "translatedText": "Hogy teljes megértsd ennek az algoritmusnak a működését csatlakozz hozzám a következő videóban, ami ennek a folytatása lesz.",
```

Reply:
Might be too long, but does sound better.